.github/workflows/switch_js_branch.yaml

```diff
@@ -0,0 +1,45 @@
+name: switch branch
```

Comment:
where is this workflow used?

Reply:
this is for switching into `releases/nightly` branch.

Reply:
We can have the logic in create_js_branch.yaml as well. I just created a separate workflow to keep it clean. Let me know if you think this is redundant.